src/helpers.js

```diff
@@ -53,6 +53,29 @@ function getDarwinApplicationVersion(bundleIdentifier) {
   return 'N/A';
 }
 
+function getAllAndroidSDKs() {
+  var androidSDKs = [];
+  try {
+    var installed = utils
+    .run(
+      '$ANDROID_HOME/tools/bin/sdkmanager --list'
+    ).split('Available')[0];
+
+    var getJustVersions = /build-tools;([\d|\.]+)[\S\s]/g;
+    while (matcher = getJustVersions.exec(installed)) {
+      androidSDKs.push(matcher[1]);
+    }
+  } catch(err) {
+    androidSDKs = ['Unknown'];
+  }
+
+  return androidSDKs;
+}
+
+function prettifyGetAllAndroidSDKs() {
```

Comment:
could we use any of the formatters for this? It occurs to me we don't have any helpers that return an array of versions yet.

Reply:
The reason I have it in a separate function is bc that's what I'll use, and I'll want the Array that is returned.

Maybe we can just have a generic "printableArray" function?  So you can have all future arrays use the same thing?

Reply:
confirmed - minor change to formatters will make this work without prettifyGetAllAndroidSDKs at all. something like:

`const version = Array.isArray(v[1]) ? v[1].join(', ') : v[1];`